modin/engines/ray/utils.py

```diff
@@ -111,11 +111,12 @@ def initialize_ray(
         elif cluster == "":
             num_cpus = os.environ.get("MODIN_CPUS", None) or multiprocessing.cpu_count()
             object_store_memory = os.environ.get("MODIN_MEMORY", None)
-            plasma_directory = None
+            plasma_directory = os.environ.get("MODIN_ON_RAY_PLASMA_DIR", None)
             if os.environ.get("MODIN_OUT_OF_CORE", "False").title() == "True":
                 from tempfile import gettempdir
```

Comment:
Please move this import under the `if plasma_directory is None` line to avoid unnecessary import.

Reply:
Done.